solr/solrj/src/java/org/apache/solr/common/cloud/SolrZkClient.java

```diff
@@ -543,6 +543,9 @@ public void makePath(String path, byte[] data, CreateMode createMode,
           throw e;
         }
       } catch (NodeExistsException e) {
+        if (log.isDebugEnabled()) {
```

Comment:
You can skip this check here, as this is parameterized and doing any string ops

Reply:
It's calling getPath which makes precommit upset, IIRC

Reply:
that's interesting.